stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1918,46 +1921,77 @@ public class ChatClient internal constructor(
         return channel(type, id)
     }
 
+    /**
+     * Creates the channel.
+     * You can either create an id-based channel by passing not blank [channelId] or
+     * member-based (distinct) channel by passing empty [channelId] and not empty [memberIds].
```

Comment:
I imagine this is a bit confusing - maybe overload the function to have either/or?

Either channelId or membersId. As this way people can send both and not be sure what happens?

Reply:
You can pass both and then you'll end up having an id-based channel with members. I'll improve KDOc